blueprints/cloud-operations/network-quota-monitoring/src/plugins/core-discover-cai-nodes.py

```diff
@@ -43,9 +43,9 @@ def _handle_discovery(resources, response, data):
       yield Resource('folders', name, {'name': result['displayName']})
     elif asset_type == 'Project':
       number = result['project'].split('/')[1]
-      data = {'number': number, 'project_id': name}
-      yield Resource('projects', name, data)
-      yield Resource('projects:number', number, data)
+      data_proj = {'number': number, 'project_id': name}
```

Comment:
there's no need to rename `data`

Reply:
can you expand on what the bug is here?

Reply:
It hides the function input parameter `data`, which I imagine is needed in L51 - `parse_page_token` will look for `pageToken` there and fail...

Reply:
ah yes sorry, data is used in the rest of the function...